frontend/src/__tests__/components/textinput.test.js

```diff
@@ -0,0 +1,188 @@
+      );
+    });
+    it('should render with a specific style on maxLength type error', () => {
+      const mockError = () => {
+        return {
+          TestInput: {
+            type: 'maxLength'
+          }
+        };
+      };
+
+      render(
+        <SingleLineInput
+          labelText={'TestInput'}
+          register={jest.fn()}
+          errors={mockError()}
+        />
+      );
+
+      const textInputComponent = screen.getByRole('textbox');
+      expect(textInputComponent).toBeInTheDocument();
+      expect(screen.getByText('TestInput')).toHaveStyle(
+        `color:${Colors.Bright[1]}`
+      );
+      expect(screen.getByText('Max length exceeded')).toBeInTheDocument();
+      expect(screen.getByText('Max length exceeded')).toHaveStyle(
+        `color:${Colors.Bright[1]}`
+      );
+    });
+  });
+  describe('Multi Line Text Input Component Tests', () => {
+    it('should render a default Multi Line Input component', () => {
+      render(<MultiLineInput register={jest.fn()} errors={jest.fn()} />);
+      const textInputComponent = screen.getByRole('textbox');
+      expect(textInputComponent).toBeInTheDocument();
+      expect(textInputComponent).toHaveStyle(
+        'width: inherit',
+        'resize: none',
+        'font-size: 1rem',
+        'border-radius: 6px',
+        `color: ${Colors.Mono[Colors.Mono.length - 1]}`,
+        `border: 1px solid ${Colors.Mono[4]}`,
+        'padding-left: 0.5rem',
+        'box-sizing: border-box'
+      );
+    });
+    it('should render a unique styled w/ Label Multi Line Input component', () => {
+      render(<MultiLineInput register={jest.fn()} errors={jest.fn()} />);
+
+      const labelComponent = screen.getByText('Label');
+      expect(labelComponent).toBeInTheDocument();
+      expect(labelComponent).toHaveStyle(
+        `color:${Colors.Mono[Colors.Mono.length - 1]}`
+      );
+
+      const textInputComponent = screen.getByRole('textbox');
+      expect(textInputComponent).toBeInTheDocument();
+      expect(textInputComponent).toHaveStyle(
+        'width: inherit',
+        'resize: none',
+        'font-size: 1rem',
+        'border-radius: 6px',
+        `color: ${Colors.Mono[Colors.Mono.length - 1]}`,
+        `border: 1px solid ${Colors.Mono[4]}`,
+        'padding-left: 0.5rem',
+        'box-sizing: border-box'
+      );
+    });
+    it('should have a label that correctly points to its input', () => {
+      render(<MultiLineInput register={jest.fn()} errors={jest.fn()} />);
+
+      const InputNode = screen.getByLabelText('Label');
+      expect(InputNode.getAttribute('name')).toBe('Label');
+    });
+    it('should show style with required error', () => {
+      const mockError = () => {
+        return {
+          Label: {
+            type: 'required'
+          }
+        };
+      };
+
+      render(<MultiLineInput register={jest.fn()} errors={mockError()} />);
+
+      const labelComponent = screen.getByText('Label');
```

Comment:
I know you named this `labelComponent` but really, `getByText` doesn't return a React Node or JSX Element.. 

nit picky, but perhaps we can name this something different given what `getByText` actually returns?

Reply:
done. Renamed to div to explain what it is capturing.

Reply:
So `screen.getByText` actually returns an element, not a 'component' - I was just double checking you were aware it doesn't actually return a JSX node